provider/datagen/Cargo.toml

```diff
@@ -75,6 +75,7 @@ itertools = "0.10"
 lazy_static = "1"
 log = "0.4"
 memchr = "2.5.0"
+once_cell = "1"
```

Comment:
@Manishearth to approve the new dependency. (We now have multiple places where it is useful)

Reply:
I'd prefer to not have _two_ lazy initialization dependencies, we should either use `lazy_static` everywhere or `once_cell` everywhere.


Slight preference for the latter since [1.70 has it in std](https://doc.rust-lang.org/stable/core/cell/struct.OnceCell.html)